src/gui/modalMenu.cpp

```diff
@@ -268,7 +268,8 @@ bool GUIModalMenu::preprocessEvent(const SEvent &event)
 			std::string label = wide_to_utf8(getLabelByID(hovered->getID()));
 			if (label.empty())
 				label = "text";
-			message += gettext(label) + ":";
+			message += gettext(label.c_str());
+			message += ":";
```

Comment:
???

Reply:
This is the code written by the original author, and I don't know why.

If there are no other problems, I will change this code back.

Reply:
This was already asked in the original PR and answered there https://github.com/minetest/minetest/pull/10019#discussion_r437763396

`message` is of type `std::string`, and the type returned by `gettext(...)` is `char*`. 
And since we are adding `":"` just after, the compiler was considering that constant also as `char*`. But adding two `char*` is not supported. 
The conversion to `std::string` comes with the `+=` assignation
So this was the simplest way to just force the conversion of the two values to `std::string`.
Note that it was not needed before because `gettext` was just a macro returning the original type that was `std::string`.

Reply:
Huh so does the libintl on other platforms have C++ support?

Reply:
I'm for it, as it seems more simple :)

Reply:
@sfan5  @oleastre Should I change the code with your style?

Reply:
Yes

Reply:
This still needs to be changed to `message += strgettext(label) + ":";`